### DISCUSSION TITLE
my account has been blocked

### DISCUSSION THREAD

**Discussion by dolardev** [community/community#134568] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone, my account was blocked a few days ago due to spam, but at no time did I spam, in fact I had almost all the codes private and the support does not answer me, any recommendations?

**Comment by LiteBrite82** [comment]
Hi @dolardev ! 
The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by dolardev** [reply]
I've been waiting for days and nothing, brother, not even a response.

**Reply by LiteBrite82** [reply]
Hi @dolardev 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

Thank you!

**Reply by dolardev** [reply]
I sent it a week ago bro

**Reply by dolardev** [reply]
?
